test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/DefaultDisplayTemplatesTest.cs

```diff
@@ -268,7 +269,7 @@ public void Display_FindsViewDataMember()
             var result = helper.Display("Property1");
 
             // Assert
-            Assert.Equal("HtmlEncode[[ViewData string]]", result.ToString());
+            Assert.Equal("HtmlEncode[[ViewData string]]", HtmlContentUtilities.HtmlContentToString(result).ToString());
```

Comment:
Don't call `ToString()` on a `string`. Any place you've chained `HtmlContentToString(blah).ToString()` is confusing and will make readers wonder about the `HtmlContentToString()` return type.